src/main.cpp

```diff
@@ -7040,9 +7043,9 @@ bool SendMessages(CNode *pto)
                 {
                     state.fSyncStarted = true;
                     state.nSyncStartTime = GetTime();
-                    state.fFirstHeadersReceived = false;
+                    // state.fFirstHeadersReceived = false;  // Only require headers within some time the first time
                     state.fRequestedInitialBlockAvailability = true;
-                    state.nFirstHeadersExpectedHeight = pindexBestHeader->nHeight;
+                    state.nFirstHeadersExpectedHeight = pindexBestHeader->nHeight-2;  // -2 to give room for slow sync
```

Comment:
What do you mean w/ "slow sync"? peers that are slow at providing or peers that are a little bit behind the actual net chaintip?